### PR TITLE
Ansible 6.0.0a3: Dependencies, changelog and porting guide

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

**Comment by dmsimard** [comment]
Setting as draft while I run sanity tests across collections (since https://github.com/ansible-community/ansible-build-data/pull/122) is still WIP)

**Review by felixfontein — APPROVED** [review]

File: 6/ansible-6.0.0a3.deps [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+cyberark.conjur: 1.1.0
+cyberark.pas: 1.0.13
+dellemc.enterprise_sonic: 1.1.0
+dellemc.openmanage: 5.3.0
+dellemc.os10: 1.1.1
+dellemc.os6: 1.0.7
+dellemc.os9: 1.0.4
+f5networks.f5_modules: 1.16.0
+fortinet.fortimanager: 2.1.5
+fortinet.fortios: 2.1.5
```
Comment (gotmax23) [inline 6/ansible-6.0.0a3.deps:64]: Isn't this the broken version?

Comment (gotmax23) [reply 6/ansible-6.0.0a3.deps:64]: FTR, I'm talking about fortinet.fortios. I [asked the maintainers](https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/commit/bbf08efdc24291665a1a506935565df53164d7dc#commitcomment-73041991) a little while ago to create a release with the fix, but they haven't done it.

Comment (felixfontein) [reply 6/ansible-6.0.0a3.deps:64]: :crying_cat_face:

Comment (dmsimard) [reply 6/ansible-6.0.0a3.deps:64]: :facepalm: you're right, it got bumped automatically, thanks for picking it up.

I'll manually pin it (again) and update the PR.

**Comment by dmsimard** [comment]
I've updated https://github.com/ansible-community/community-topics/issues/96 after running sanity tests, we're down to 51 collections failing sanity tests with 6.0.0a3 (from 59 in 6.0.0a2).

File: 6/ansible-6.0.0a3.deps (hunk 1 above)
Comment (dmsimard) [reply 6/ansible-6.0.0a3.deps]: Fixed in https://github.com/ansible-community/ansible-build-data/pull/123/commits/165555a57771f2e1808248bc590b1afb688bd3bc

**Comment by dmsimard** [comment]
It's getting kind of late, I'll delay the release until tomorrow morning (EST).

**Comment by dmsimard** [comment]
Thanks for reviewing, it's good go to.

**Pull request merged**